src/libstd/process.rs

```diff
@@ -134,6 +134,16 @@ use crate::sys_common::{AsInner, AsInnerMut, FromInner, IntoInner};
 /// the parent process wait until the child has actually exited before
 /// continuing.
 ///
+/// # Warning
+///
+/// On some system, calling [`wait`] or similar is necessary for the OS to release resources. A
+/// process that terminated but has not been waited on is still around as a „zombie“. Leaving too
+/// many zombies around may exhaust global resources (for example process IDs).
+///
+/// The standard library does *not* automatically wait on child processes (not even if the `Child`
+/// is dropped), it is up to the application developer to do so. As a consequence, dropping `Child`
+/// handles without waiting on them first is not recommended in long-running applications.
+///
```

Comment:
Those lines in this paragraph is a little wide compare to surround ones.

Reply:
I've my editor set to 120 columns for Rust, which is AFAIK the officially recommended value. My guess is the surrounding text predates this recommendation and is simply narrower as opposed to my new text wider 😇.

Anyway, if being consistent with the rest is preferred, I'm ok with making the new text narrower too.

Reply:
I checked some new-ish files (related to futures and tasks) and things seem to be generally kept around 80 columns, so I would propose to follow that here as well.